lighthouse-core/audits/seo/font-size.js

```diff
@@ -24,6 +24,10 @@ const UIStrings = {
   /** Explanatory message stating that there was a failure in an audit caused by a missing page viewport meta tag configuration. "viewport" and "meta" are HTML terms and should not be translated. */
   explanationViewport: 'Text is illegible because there\'s no viewport meta tag optimized ' +
     'for mobile screens.',
+  /** Label for the table row which summarizes all failing nodes that were not fully analyzed. This text overrides the source location field. "Add'l" is shorthand for "Additional" */
```

Comment:
nit: can probablyremove "this text overrides...", translators won't have that level of visibility